post-scan-actions/azure-python-slack-notification/template.json

```
@@ -0,0 +1,233 @@
+                "enabled": true,
+                "reserved": true,
+                "isXenon": false,
+                "hyperV": false,
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+                "siteConfig": {
+                    "use32BitWorkerProcess": false,
+                    "linuxFxVersion": "PYTHON|3.8",
+                    "appSettings": [{
+                            "name": "AzureWebJobsStorage",
+                            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccountName'), ';EndpointSuffix=', environment().suffixes.storage, ';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value)]"
+                        },
+                        {
+                            "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+                            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', variables('storageAccountName'), ';EndpointSuffix=', environment().suffixes.storage, ';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value)]"
+                        },
+                        {
+                            "name": "WEBSITE_CONTENTSHARE",
+                            "value": "[toLower(variables('functionAppName'))]"
+                        },
+                        {
+                            "name": "FUNCTIONS_EXTENSION_VERSION",
+                            "value": "~3"
+                        },
+                        {
+                            "name": "WEBSITE_NODE_DEFAULT_VERSION",
+                            "value": "~10"
+                        },
+                        {
+                            "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                            "value": "[reference(resourceId('microsoft.insights/components', variables('applicationInsightsName')), '2020-02-02-preview').InstrumentationKey]"
+                        },
+                        {
+                            "name": "FUNCTIONS_WORKER_RUNTIME",
+                            "value": "python"
+                        },
+                        {
+                            "name": "SCAN_RESULT_TOPIC_SUBSCRIPTION",
+                            "value": "[variables('scanResultTopicSubscriptionName')]"
+                        },
+                        {
+                            "name": "SLACK_URL_WEBHOOK",
+                            "value": "[variables('slackWebHook')]"
+                        },
+                        {
+                            "name": "SLACK_CHANNEL",
+                            "value": "[variables('slackChannel')]"
+                        },
+                        {
+                            "name": "SUB_NAME",
+                            "value": "[variables('subscriptionName')]"
```

Subscription name can be retrieved by template function.
https://docs.microsoft.com/zh-tw/azure/azure-resource-manager/templates/template-functions-resource#subscription
Then `subscriptionName` variable and parameter can be removed.
```suggestion
                            "value": "[subscription().displayName]"
```